src/components/AsyncTable/RowPlaceholders.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { RowPlaceholders } from './Placeholder';
+import { StoryCard } from '../.storybook';
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+
+export default {
+  title: 'Placeholder Elements/RowPlaceholders',
+  component: RowPlaceholders
+} as ComponentMeta<typeof RowPlaceholders>;
+
+const Template: ComponentStory<typeof RowPlaceholders> = args => {
+  return (
+    <StoryCard isContrast>
+      <RowPlaceholders {...args} />
```

Comment:
The following error is displayed when running in storybook, however, after some investigation this error is not thrown when running a zApp using the Table component. I believe this may be something to do with Storybook rendering the components wrapped in div tags? Either way this shouldn't occur when using these components in zApps.

<img width="570" alt="Screenshot 2023-02-20 at 15 53 43" src="https://user-images.githubusercontent.com/39112648/220012445-4082db6a-0735-4a97-b954-fdfd144b143f.png">